core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -702,6 +702,13 @@ trait DeltaSQLConfBase {
         .checkValue(_ > 0, "'optimize.maxThreads' must be positive.")
         .createWithDefault(15)
 
+  val DELTA_OPTIMIZE_USE_REPARTITON =
+    buildConf("optimize.repartition.enabled")
```

Comment:
nit: i think it will be better to name this `optimize.useRepartition.enabled`. consistent with `DELTA_OPTIMIZE_USE_REPARTITON` variable name as well.

Reply:
If so I would change the variable name to DELTA_OPTIMIZE_REPARTITION_ENABLED ? 
or how about `optimize.compact.useRepartition` ? as it's only for compaction